Pcap++/header/PcapRemoteDeviceList.h

```diff
@@ -85,29 +92,34 @@ namespace pcpp
 		 * @param[in] ipAddress The IP address of the remote machine through which clients can connect to the rpcapd
 		 * daemon
 		 * @param[in] port The port of the remote machine through which clients can connect to the rpcapd daemon
-		 * @return A smart pointer to the newly created PcapRemoteDeviceList, or nullptr if (an appropriate error will be printed
-		 * to log in each case):
+		 * @return A smart pointer to the newly created PcapRemoteDeviceList, or nullptr if (an appropriate error will
+		 * be printed to log in each case):
 		 * - WinPcap/Npcap encountered an error in creating the remote connection string
 		 * - WinPcap/Npcap encountered an error connecting to the rpcapd daemon on the remote machine or retrieving
 		 * devices on the remote machine
 		 */
 		static std::unique_ptr<PcapRemoteDeviceList> createRemoteDeviceList(const IPAddress& ipAddress, uint16_t port);
 
 		/**
-		 * An overload of the previous getRemoteDeviceList() method but with authentication support. This method is suitable for connecting to
-		 * remote daemons which require authentication for accessing them
-		 * @param[in] ipAddress The IP address of the remote machine through which clients can connect to the rpcapd daemon
-		 * @param[in] port The port of the remote machine through which clients can connect to the rpcapd daemon
-		 * @param[in] remoteAuth A pointer to the authentication object which contains the username and password for connecting to the remote
+		 * An overload of the previous getRemoteDeviceList() method but with authentication support. This method is
+		 * suitable for connecting to remote daemons which require authentication for accessing them
+		 * @param[in] ipAddress The IP address of the remote machine through which clients can connect to the rpcapd
 		 * daemon
-		 * @return A pointer to the newly created PcapRemoteDeviceList, or NULL if (an appropriate error will be printed to log in each case):
+		 * @param[in] port The port of the remote machine through which clients can connect to the rpcapd daemon
+		 * @param[in] remoteAuth A pointer to the authentication object which contains the username and password for
+		 * connecting to the remote daemon
+		 * @return A pointer to the newly created PcapRemoteDeviceList, or NULL if (an appropriate error will be printed
+		 * to log in each case):
 		 * - IP address provided is NULL or not valid
 		 * - WinPcap/Npcap encountered an error in creating the remote connection string
-		 * - WinPcap/Npcap encountered an error connecting to the rpcapd daemon on the remote machine or retrieving devices on the remote machine
-		 * @deprecated This factory function has been deprecated in favor of 'createRemoteDeviceList' factory for better memory safety.
+		 * - WinPcap/Npcap encountered an error connecting to the rpcapd daemon on the remote machine or retrieving
+		 * devices on the remote machine
```

Comment:
Add extra spaces to align with bullet point

Reply:
Fixed 61ac4b8